File: tugas/python/Pc.py
from Disk import Disk
from Processor import Processor
from Ram import Ram

class Pc():
    # constructor
    def __init__(self, disk=Disk(), ram=Ram(), processor=Processor(), totalPrice=0):
        self.__disk = disk
        self.__ram = ram
        self.__processor = processor
        self.__totalPrice = totalPrice

    # setter n getter
    def setDisk(self, disk):
        self.__disk = disk

    def getDisk(self):  
        return self.__disk

    def setRam(self, ram):
        self.__ram = ram

    def getRam(self):
        return self.__ram

    def setProcessor(self, processor):
        self.__processor = processor

    def getProcessor(self):
        return self.__processor

    def setTotalPrice(self, totalPrice):
        self.__totalPrice = totalPrice

    def getTotalPrice(self):
        return self.__totalPrice

File: tugas/php/Pc.php
<?php

include "Disk.php";
include "Processor.php";
include "Ram.php";

class Pc{
    // atribut
    private $disk, $ram, $processor, $totalPrice;

    // constructor
    public function __construct(){
        $this->disk = new Disk();
        $this->ram = new Ram();
        $this->processor = new Processor();
        $this->totalPrice = 0;
    }

    // setter n getter
    public function setDisk($disk){
        $this->disk = $disk;
    }

    public function getDisk(){
        return $this->disk;
    }
    
    public function setRam($ram){
        $this->ram = $ram;
    }

    public function getRam(){
        return $this->ram;
    }

    public function setProcessor($processor){
        $this->processor = $processor;
    }

    public function getProcessor(){
        return $this->processor;
    }

    public function setTotalPrice($totalPrice){
        $this->totalPrice = $totalPrice;
    }

    public function getTotalPrice(){
        return $this->totalPrice;
    }
}